Changelog.md

```diff
@@ -1,3 +1,7 @@
+# 0.8.4
+
+- Add function to get used and free heap size ([#32](https://github.com/phil-opp/linked-list-allocator/pull/32))
+
```

Comment:
Thanks for directly adding a changelog entry, this is useful! Could you remove the `# 0.8.4` line since I normally do the version increment in a separate commit?

Reply:
Done.